packages/gatsby/src/schema/graphql-engine/shims/gatsby-sharp.js

```diff
@@ -0,0 +1,8 @@
+import sharp from "sharp"
```

Comment:
not sure if importing `sharp` here is correct. `gatsby` doesn't depend on it, but this module should be alias for `gatsby-sharp`, so not sure now if context of imports here is `gatsby` or `gatsby-sharp`

Reply:
Not 100% sure what you mean here. Gatsby-sharp is an async wrapper for requiring sharp

Reply:
I mean this file is in `gatsby` which doesn't have `sharp` as dep, but is `gatsby-sharp` dep, so not sure if I can just import it here like that

Reply:
Probably not in PNP world, what we can do is re-exports plain sharp form gatsby-sharp/sharp

Reply:
Done in [gatsbyjs/gatsby@`74bc1ad` (#36534)](https://github.com/gatsbyjs/gatsby/pull/36534/commits/74bc1ad2cc1ff3469f5628e2fabd9a4c38d451e4)